dotnet/src/webdriver/Internal/Logging/LogContext.cs

```diff
@@ -56,14 +58,11 @@ public LogContext(LogEventLevel level, ILogContext parentLogContext, ConcurrentD
             }
         }
 
-        public ILogContext CreateContext()
-        {
-            return CreateContext(_level);
-        }
+        public ILogContext CreateContext() => CreateContext(_level);
```

Comment:
Please don't use it at this time for methods: I already broke rules for new code like `_field` instead of `this.field`, or `is` instead of `==`. You understand why.

Reply:
Reverted.

> like `_field` instead of `this.field`

I see now that the `_` is not the preferred convention. I will make that change to the editorconfig PR, so there will be no more concerns about style.